### DISCUSSION TITLE
Bug/Request

### DISCUSSION THREAD

**Discussion by liamamiller** [CircuitSetup/Time-Circuits-Display#37] [Ideas]
Hi all! Wanted to point out a bug I've come across...

If I press and hold "4" to turn off the displays, it only turns off the displays for about 10 minutes. I haven't managed to time this. I'm guessing it re-sets after the sketch loops? Either way, the display off schedule feature works properly. Just seems to be the manual override.

I'd also like to implement Destination Time and Last Time Departed to show chosen time zones. For example, London, England and Sydney Australia.

Loving my time circuits!

**Comment by CircuitSetup** [comment]
4 toggles night mode. What do you have this set to in the settings?

Different time zones is a cool idea. Would the time zone be displayed instead of the date for last time departed and destination times?

**Reply by liamamiller** [reply]
> 4 toggles night mode. What do you have this set to in the settings?
> 
> Different time zones is a cool idea. Would the time zone be displayed instead of the date for last time departed and destination times?

Hey John - the settings have night mode on from 1a to 8a and works flawlessly. If I press and hold 4 to manually turn on night mode (say, if I'm watching a movie) the displays turn off and then turn themselves back on after about 10 minutes.

RE time zones - either displaying the time zone in the date or just showing the date in that time zone works. It would just be cool to look up and see the time in my family's timezone!

**Comment by doc1011** [comment]
I think time zones would be a great feature.I like the idea with destination and last time departed, with the RTC in the middle as it is now. That way you could use the Month section for a city code with the relevant timezone.

You could also use the YEAR to display timezones across destination/present/last (if digits allow?). Being able to switch to time zone mode and seeing MONTH / DAY / ZONE / HH:MM like with sensor data would be really cool.

**Comment by alalal80** [comment]
Good morning,
I already congratulate you for your excellent work.
everything is well thought out.
I also had this idea world clock.
but I will leave the month but I will alternate the display with the chosen city.
all configurable in the web application
the city and the alternating time.
an other idea.
have the possibility of having a countdown for a given date and time.
Well, thank you again for your work.

**Reply by alalal80** [reply]
Good evening,
another idea for the alarm,
why not add an output that will control a servo motor to ring the ticks of the clock instead of the sound of the clock.
still great work.

**Comment by realA10001986** [comment]
The manual override timeout is intentional and documented. It's 30 minutes. After that, a sensor and/or a programmed schedule takes over again.

World Clock mode is already implemented upstream.
